[2024-12-06T15:48:51.486+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T15:48:51.497+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_cleaned_data.set_vars manual__2024-12-06T15:47:22.812208+00:00 [queued]>
[2024-12-06T15:48:51.503+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_cleaned_data.set_vars manual__2024-12-06T15:47:22.812208+00:00 [queued]>
[2024-12-06T15:48:51.504+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-12-06T15:48:51.514+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): set_vars> on 2024-12-06 15:47:22.812208+00:00
[2024-12-06T15:48:51.519+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3305) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T15:48:51.521+0000] {standard_task_runner.py:72} INFO - Started process 3306 to run task
[2024-12-06T15:48:51.521+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'insert_cleaned_data', 'set_vars', 'manual__2024-12-06T15:47:22.812208+00:00', '--job-id', '2037', '--raw', '--subdir', 'DAGS_FOLDER/insert_cleaned_data.py', '--cfg-path', '/tmp/tmpl_wswwmw']
[2024-12-06T15:48:51.522+0000] {standard_task_runner.py:105} INFO - Job 2037: Subtask set_vars
[2024-12-06T15:48:51.552+0000] {task_command.py:467} INFO - Running <TaskInstance: insert_cleaned_data.set_vars manual__2024-12-06T15:47:22.812208+00:00 [running]> on host cb0074367deb
[2024-12-06T15:48:51.609+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='artzsh' AIRFLOW_CTX_DAG_ID='insert_cleaned_data' AIRFLOW_CTX_TASK_ID='set_vars' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T15:47:22.812208+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-06T15:47:22.812208+00:00'
[2024-12-06T15:48:51.610+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T15:48:51.635+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-12-06T15:48:51.639+0000] {insert_cleaned_data.py:106} INFO - Обновлён dm_var до 7
[2024-12-06T15:48:51.640+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-06T15:48:51.646+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T15:48:51.646+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=insert_cleaned_data, task_id=set_vars, run_id=manual__2024-12-06T15:47:22.812208+00:00, execution_date=20241206T154722, start_date=20241206T154851, end_date=20241206T154851
[2024-12-06T15:48:51.695+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T15:48:51.709+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-06T15:48:51.711+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
